recipe/meta.yaml

```diff
@@ -40,7 +40,8 @@ requirements:
     # the "torch" extra by default in the runtime requirements. So we
     # can't remove that dependency without breaking all our users.
     - pytorch >=1.9,<3
-    - lightning >=2.0,<2.2
+    # Relax upperbound to match autogluon-timeseries
+    - lightning >=2.0,<2.4
     # Capping `lightning` does not cap `pytorch_lightning`, so we cap manually
     # Relax upperbound to match autogluon-timeseries
     - pytorch-lightning >=2.0,<2.4
```

Comment:
Should we indicate which version of `autogluon.timeseries`'s dependencies we are relaxing?
I get the impression that 1.1.1 sets a minimum bound of 2.2 for `*lightning` so we're not relaxing _vis-a-vis_ that version, for example.

Reply:
We are relaxing for `autogluon 1.1.1` since it sets `>=2.2` which is incompatible with the original `<2.2` here. I agree that it's a good idea to mention the version here, adding it.

Reply:
Why does the comment say `autogluon-timeseries` if it is for `autogluon`?

(Also, why is it spelt `autogluon-timeseries` rather than `autogluon.timeseries`?)

Reply:
My mistake, I meant to say `autogluon.timeseries 1.1.1` in my previous comment, I used `autogluon` as a shorthand. I corrected the package name in the latest commit [here](https://github.com/AnacondaRecipes/gluon-ts-feedstock/pull/5/commits/b724ef2f852c708da31425835ead60666778f3e6).